src/main/java/graphql/execution/ExecutionContextBuilder.java

```diff
@@ -172,6 +172,11 @@ public ExecutionContextBuilder valueUnboxer(ValueUnboxer valueUnboxer) {
         return this;
     }
 
+    public ExecutionContextBuilder resetErrors() {
+        this.errors.clear();
+        return this;
+    }
```

Comment:
we need this because in subscroptions we start a new execution per streamed event